src/patterns/dosdp-patterns/response_to_chemical_with_role_stimulus.yaml

```diff
@@ -0,0 +1,54 @@
+---
+pattern_name: response_to_chemical_with_role_stimulus
+
+pattern_iri: http://purl.obolibrary.org/obo/oba/patterns/response_to_chemical_with_role_stimulus.yaml
+
+description: "A trait that affects the biological response to a representative or representatives of a chemical entity group where the members of the group share a role which they can exhibit. Examples of roles include 'drug', 'protease inhibitor', 'opioid agent', etc."
+
+contributors:
+  - https://orcid.org/0000-0001-8314-2140  # Ray Stefancsik
+
+classes:
+  role: CHEBI:50906
+  process_quality: PATO:0001236
+  response_to_stimulus: GO:0050896
+
+
+relations:
+  characteristic_of: RO:0000052
+  has_input: RO:0002233
+
+annotationProperties:
+  exact_synonym: oio:hasExactSynonym
+  contributor: dcterms:contributor
+
+vars:
+  role: "'role'"
+
+data_vars:
+  orcid: "'contributor'"  # ORCID of contributor
+
+name:
+  text: "trait in response to %s"
+  vars:
+    - role
+
+annotations:
+  - annotationProperty: exact_synonym
+    text: "response to %s trait"
+    vars:
+      - role
+  - annotationProperty: contributor
+    text: "%s"
+    vars:
+      - orcid
+
+def:
+  text: "A trait that affects the response to a stimulus with %s."
+  vars:
+    - role
+
+equivalentTo:
+  text: "'process_quality' and 'characteristic_of' some ('response_to_stimulus' and 'has_input' some %s)"
```

Comment:
Does this reflect how GO does it?

I would prefer additional redundant () wherever possible, so 

```
('response_to_stimulus' and ('has_input' some %s))
```

Reply:
Done.